index.rst

```diff
@@ -1,29 +1,57 @@
-:tocdepth: 1
+:tocdepth: 2
 
 .. sectnum::
 
 .. Metadata such as the title, authors, and description are set in metadata.yaml
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
+.. warning::
 
    **This technote is a work-in-progress.**
 
 Abstract
 ========
 
-This technote summarizes the groups and individuals making in-kind contributions to SIT-Com
+This technote summarizes the groups and individuals making in-kind contributions to the Vera C. Rubin Observatory System Integration, Test, and Commissioning (SIT-Com) effort.
+
+Introduction
+============
+
+The Vera C. Rubin Observatory recognizes the individuals listed below making value-added non-financial contributions to the System Integration, Test, and Commissioning (SIT-Com) effort in order to facilitate an efficient transition to LSST Operations.
+By sharing their technical and scientific expertise, these individuals enhance and diversify the Project's planned commissioning effort.
+The named participants will work directly alongside Rubin Observatory staff and will have access to commissioning data products as they are acquired in order to complete their assignments.
+
+The participating individuals were invited to join the Rubin Observatory SIT-Com effort through one of two programs:
+
+1. US/Chile Commissioning Announcement of Opportunity Program and
+2. International In-Kind Contribution Program where the recipient group is the “Rubin Commissioning Team”.
+
+The two types of contribution are being managed by Rubin Observatory as a combined resource.
```

Comment:
Consider:
Both types of contributions are...

(either way you need to make contribution plural)

Reply:
Done